skabase/SKABaseDevice/SKABaseDevice.py

```diff
@@ -50,15 +48,18 @@ class SKABaseDevice(with_metaclass(DeviceMeta, Device)):
     # PROTECTED REGION ID(SKABaseDevice.class_variable) ENABLED START #
 
     def _init_syslog(self):
+        self.formatter = logging.Formatter('%(name)s: %(levelname)s %(module)s %(message)r')
         try:
             self.logger = logging.getLogger(__name__)
-            #self.syslogs = SysLogHandler(address='/dev/log', facility='syslog')
+            # self.syslogs = SysLogHandler(address='/dev/log', facility='syslog')
             self.syslogs = SysLogHandler(address='/var/run/rsyslog/dev/log', facility='syslog')
-            self.formatter = logging.Formatter('%(name)s: %(levelname)s %(module)s %(message)r')
             self.syslogs.setFormatter(self.formatter)
             self.logger.addHandler(self.syslogs)
-        except Exception:
-            self.error_stream("Syslog cannot be initialized:")
+        except EnvironmentError as ex:
+            log_file_name = self.__class__.__name__
```

Comment:
wouldn't it be better to call with the device name? What happen if you have two devices with the same class name?

Reply:
Replaced the class name with device name.